overlord/snapstate/export_test.go

```diff
@@ -471,3 +471,47 @@ func MockCgroupMonitorSnapEnded(f func(string, chan<- string) error) func() {
 		cgroupMonitorSnapEnded = old
 	}
 }
+
+func CalculateTaskProgress(st *state.State, snapName string, instanceName string, revision snap.Revision) (int, int, string) {
+	return calculateTaskProgress(st, snapName, instanceName, revision)
+}
+
+func NotifyBeginRefresh(st *state.State, taskCount int, instanceName string, revision snap.Revision, snapName string, desktopPath string) {
+	notifyBeginRefresh(st, taskCount, instanceName, revision, snapName, desktopPath)
+}
+
+func SnapSetupAndState(t *state.Task) (*SnapSetup, *SnapState, error) {
+	return snapSetupAndState(t)
+}
```

Comment:
These can be simplified

```suggestion
var (
    CalculateTaskProgress = calculateTaskProgress
    NotifyBeginRefresh = notifyBeginRefresh
    SnapSetupAndState = snapSetupAndState
)
```